MekHQ/src/mekhq/gui/dialog/CustomizeAtBContractDialog.java

```diff
@@ -188,8 +188,10 @@ private void initComponents() {
     	btnEnemyCamo = new JButton();
         JLabel lblEnemyRating = new JLabel();
     	JLabel lblRequiredLances = new JLabel();
-    	spnRequiredLances = new JSpinner(new SpinnerNumberModel(contract.getRequiredLances(), 1,
-    			null, 1));
+    	
+    	int requiredLances = contract.getRequiredLances() > 0 ? contract.getRequiredLances() : 1; 
```

Comment:
You may want to call `mekhq.campaign.mission.AtBContract.calculatePaymentMultiplier(Campaign)` instead of hardcoding a minimum of 1 lance.

For reference, this is the commit that solved the issue for the contract market dialog (which does call `calculatePaymentMultiplier()` in its `initComponents()`)  https://github.com/MegaMek/mekhq/commit/3337b64006efd2a71f79d53d75cbe8395c3a8e9b

Reply:
I see your point, but I'm not sure that I want to re-calculate all the contract details in this situation. For better or worse, the contract was signed based on a particular state of the player force, and it may not be a pleasant experience to suddenly have the contract details (such as payment) change suddenly after it is signed.

Reply:
Good point.